docs/guides/latest/volumesnapshot/statefulset.md

```diff
@@ -0,0 +1,828 @@
+
+### Restore same data in all replica
+
+This section will show you how to restore PVCs from the snapshot that  we have taken in earlier section.
+
+#### Create RestoreSession:
+
+At first, we have to create a `RestoreSession` crd to restore the PVCs from respective snapshot.
+
+Below is the YAML of the `RestoreSesion` crd that we are going to create,
+
+```yaml
+apiVersion: stash.appscode.com/v1beta1
+kind: RestoreSession
+metadata:
+  name: restore-pvc
+  namespace: demo
+spec:
+  driver: VolumeSnapshotter
+  target:
+    replicas : 3
+    volumeClaimTemplates:
+      - metadata:
+          name: source-data-stash-demo
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "standard"
+          resources:
+            requests:
+              storage: 6Gi
+          dataSource:
+            kind: VolumeSnapshot
+            name: ${CLAIM_NAME}-0-1560231666
+            apiGroup: snapshot.storage.k8s.io
+```
+Here,
+
+
+* `spec.dataSource.name`: `spec.dataSource.name` is the `VolumeSnapshot` resource name. `$(CLAIM_NAME)` is resolved by the stash operator or replaced by the `metadata.name` of the `volumeclaimtemplates`.
+    
+Let's create the `BackupConfiguration` crd we have shown above.
+
+```console
+$ kubectl create -f ./docs/examples/volume-snapshot/statefulset/restoresession.yaml
+restoresession.stash.appscode.com/restore-pvc created
+````
+Once, you have created the `RestoreSession` crd, Stash will create a job to restore. We can watch the `RestoreSession` phase to check if the restore process is succeeded or not.
+
+Run the following command to watch RestoreSession phase,
+
+```console
+$ watch -n 1 kubectl get restore -n demo
+Every 1.0s: kubectl get restore -n demo                      suaas-appscode: Tue Jun 18 18:35:41 2019
+
+NAME          REPOSITORY-NAME   PHASE       AGE
+restore-pvc                     Running     10s
+restore-pvc                     Succeeded   1m
+```
+So, we can see from the output of the above command that the restore process succeeded.
+
+##### Verify Restored PVC:
+
+Once a restore process is complete, we will see that new PVCs with the name `source-pvc-stash-demo-0` , `source-pvc-stash-demo-1` and `source-pvc-stash-demo-2` has been created successfully.
+
+check that the status of the PVCs are bound,
+
+```console
+$  kubectl get pvc -n demo
+NAME                       STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+source-data-stash-demo-0   Bound    pvc-7c773921-8c10-11e9-bd3e-42010a800011   6Gi        RWO            standard       10m
+source-data-stash-demo-1   Bound    pvc-7c7b32ad-8c10-11e9-bd3e-42010a800011   6Gi        RWO            standard       10m
+source-data-stash-demo-2   Bound    pvc-7c7d0bdf-8c10-11e9-bd3e-42010a800011   6Gi        RWO            standard       10m
+````
+
+##### Verify Restored Data:
+
+We will create a new Statefulset to verify whether the restored data has been restored successfully. 
```

Comment:
We will create a new Statefulset with the restored PVCs to verify whether the backed up data has been restored.